jupyter-notebooks/udm/udm.ipynb

```diff
@@ -0,0 +1,434 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with the Unusable Data Mask (UDM)\n",
+    "\n",
+    "## Introduction\n",
+    "The [Unusable Data Mask (UDM) File](https://www.planet.com/docs/spec-sheets/sat-imagery/#72-unusable-data-mask-file) is provided with many Planet products. It provides pixel-by-pixel information on quality issues that may affect the usability of each pixel, depending on the application.\n",
+    "\n",
+    "The UDM is an 8-bit single-band image. Pixel quality issues are bit-encoded. The bit values are given in the [UDM description](https://www.planet.com/docs/spec-sheets/sat-imagery/#72-unusable-data-mask-file). For example, if Bit 0 is set to 1 for a pixel in the UDM, then the pixel is in an area that was not imaged by the spacecraft, and the pixel is 'blackfill'. If all bits are set to zero, there is no quality issue with the pixel. \n",
```

Comment:
Thanks for expanding the definition here and giving more context. Once I revise the glossary in the docs, I'll add links from `blackfill` `single-band` etc.